.github/workflows/coverage-gh-pages.yml

```diff
@@ -33,11 +33,11 @@ jobs:
       - name: Install dependencies
         run: sudo apt install -y ninja-build
       - name: Configure
-        run: mkdir build && cd build && cmake -G Ninja -DLLVM_PARALLEL_LINK_JOBS=1 -DLLVM_USE_LINKER=lld -DCMAKE_BUILD_TYPE=Release -DDXC_COVERAGE=On -C ${{github.workspace}}/cmake/caches/PredefinedParams.cmake -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ ${{github.workspace}}
+        run: mkdir build && cd build && cmake -G Ninja -DLLVM_PARALLEL_LINK_JOBS=1 -DLLVM_USE_LINKER=lld -DCMAKE_BUILD_TYPE=Release -DDXC_USE_LIT=On -DDXC_COVERAGE=On -C ${{github.workspace}}/cmake/caches/PredefinedParams.cmake -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ ${{github.workspace}}
```

Comment:
I see a lot of `cd build && ninja ...`, they should work with `ninja -C build ...` no?

Reply:
Yea, I could also use `cmake -B <>` to remove the `mkdir && cd` there too.

Reply:
Updated.